compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1121,6 +1121,9 @@ warning.usage.of.deprecated.construct=\
 warning.non.module.qualified.error.reason=\
   error reason ''{0}'' is not module qualified
 
+warning.function.does.not.return=\
+  This function doesn't return, hence use 'returns never' here
```

Comment:
I think a better mesaage would be missing ‘returns never’ in the definition of function ‘’{0}’’. WDYT?

Reply:
+adding
- I think it is a practice not to use short forms like "doesn't"
- does not start with a capital letter
- being a warning rather than an error type, the message looks a bit forceful. I believe the idea of giving a warning is maybe because the user forgot to add `returns never`. It can also be the case that this is not intentional. So the warning should guide the user to find the actual solution